### ISSUE TITLE
Suggestion: switching between beep and buzz

### ISSUE THREAD

**Issue by zloid** [Weiming-Hu/hourstrike#3]
It would be great to be able to switch between beep sound and vibration just like in the app Hour Chime.

**Comment by Weiming-Hu** [comment]
Good idea. This would probably be a quick feature addition. Let me take a look.
